File: mvp.admin/src/FlexForm.Api.Admin/Model/FormControlBindingModel.cs
namespace FlexForm.Api.Admin.Model
{
    public class FormControlBindingModel : ModelBaseInt
    {
        public int FieldDefinitionId { get; set; }

        public bool IsReadOnly { get; set; }

        public bool IsInvalidStateEnabled { get; set; }

        public dynamic UiProperties { get; set; }
    }
}

File: mvp.admin/src/FlexForm.Admin/flexform-admin/src/app/models/dtos/form-control-binding.model.ts
import { FieldDefinition } from './field-definition.model';
import { FormControl } from '@angular/forms';
import { BaseProperty } from '../properties/base-property.model';
import { UiPropertiesInterface } from '../interfaces/ui-properties.interface';
import { BaseWithTimeStamp } from './base-with-timestamp.model';

export class FormControlBinding extends BaseWithTimeStamp<number> implements UiPropertiesInterface{

    fieldDefinition:FieldDefinition;
    fieldDefinitionId:number;
    isReadOnly:boolean;
    isInvalidStateEnabled:boolean;
    uiProperties: BaseProperty<any>[];
    formControl:FormControl;
    
}